pkg/prowgen/prowgen.go

```diff
@@ -269,13 +288,17 @@ func GenerateJobs(configSpec *cioperatorapi.ReleaseBuildConfiguration, info *Pro
 }
 
 func generateCiOperatorPodSpec(info *ProwgenInfo, secrets []*cioperatorapi.Secret, targets []string, additionalArgs ...string) *corev1.PodSpec {
+	var needHiveKubeconfig bool
```

Comment:
What I meant was: Remove the boolean arg from everywhere, including `generatePodSpec`. Just add the secret to the `secrets` slice. Make the ci-operator not need a flag but derive from the steps if it needs a hive kubeconfig. Default the location for the hive kubeconfig. If the ci-operator derived from the steps that it needs the hive kubeconfig, it should look at the default location and fail if its not present there.

Reply:
Updated.